datahub-graphql-core/src/main/java/com/linkedin/datahub/graphql/GmsGraphQLEngine.java

```diff
@@ -895,6 +962,15 @@ private void configureGlossaryRelationshipResolvers(final RuntimeWiring.Builder
         );
     }
 
+    private void configureDomainResolvers(final RuntimeWiring.Builder builder) {
+        builder.type("Domain", typeWiring -> typeWiring
+            .dataFetcher("entities", new DomainEntitiesResolver(this.entityClient))
+            .dataFetcher("relationships", new AuthenticatedResolver<>(
+                new EntityRelationshipsResultResolver(graphClient)
+            ))
```

Comment:
can you say more about domains having relationships? Is that something that currently exists or is this future proofing?

Reply:
resolved offline!